proto/kava/community/v1beta1/params.proto

```diff
@@ -12,4 +13,6 @@ message Params {
     (gogoproto.stdtime) = true,
     (gogoproto.nullable) = false
   ];
+
+  cosmos.base.v1beta1.Coin rewards_per_second = 2 [(gogoproto.nullable) = false];
```

Comment:
do we even want to support minting non-ukava coins? If not, perhaps its better to just make this an int?

Reply:
Yup we can use an int instead

Reply:
Agree with using `int` here -- though should we rename this paramter to reflect that? `StakingRewardsPerSecond` or something to make it clear this is what the instended use is and thus using the staking denom?

Reply:
I also don't like using the same parameter name across modules with different types.  RewardsPerSecond is coins in other places but int here

Reply:
So imo a re-name also makes sense but open to other thoughts as well

Reply:
`StakingRewardsPerSecond` sounds good to be more clear